tests/integration/IntegrationTests.cmake

```diff
@@ -27,6 +27,7 @@ foreach(test ${INTEGRATION_TESTS_LIST})
 		set(SET_TEST_PROPS
 	"set_tests_properties([==[${test}]==] PROPERTIES
 		WORKING_DIRECTORY \"${CMAKE_CURRENT_SOURCE_DIR}\"
+		TIMEOUT 15
```

Comment:
```suggestion
		TIMEOUT 60
```

If tests finish in 10 seconds on a typical day, then I like to have a bit of safety margin for a bad day.

Reply:
I do agree that 1500 seconds is too long.

Reply:
30 seconds might also be okay.

Reply:
```suggestion
		TIMEOUT 30
```